src/utils/matmul_helper.h

```diff
@@ -93,6 +93,113 @@ class MMHelper {
     //               transposed                          not_transposed
     //
 
+// need revise in xdnn
+#define BLOCK_N 64
+    void xdnn_hgemm_f32f16f32_packb(bool transB, int N, int K, const XDNN_FP16 *B, int ldb, XDNN_FP16 *packedB) {
```

Comment:
move into xDNN

Reply:
Do you want to put to 'kernels' directory in current version?

Reply:
This is Wenhuan's PR. Wenhuan will create a new PR.